src/xunit.performance.api/Model/AssemblyModel.cs

```diff
@@ -175,6 +175,143 @@ private static IEnumerable<(string testCaseName, string metric, IEnumerable<doub
+            using (var stream = File.Create(xmlFileName))
+            {
+                using (var sw = new StreamWriter(stream))
+                {
+                    new XmlSerializer(typeof(BenchmarkScenario))
+                        .Serialize(sw, this, namespaces);
+                }
+            }
+        }
+
+        internal DataTable GetStatistics()
+        {
+            var dt = new DataTable();
+            var col0_testName = dt.AddColumn("Test Name");
+            var col1_metric = dt.AddColumn("Metric");
+            var col2_iterations = dt.AddColumn("Iterations");
+            var col3_average = dt.AddColumn("AVERAGE");
+            var col4_stdevs = dt.AddColumn("STDEV.S");
+            var col5_min = dt.AddColumn("MIN");
+            var col6_max = dt.AddColumn("MAX");
+
+            foreach (var asm in Assemblies)
+            {
+                foreach (var test in asm.Tests)
+                {
+                    foreach (var metric in test.Performance.Metrics)
+                    {
+                        var values = test.Performance.IterationModels
+                            .Where(iter => iter.Iteration.ContainsKey(metric.Name))
+                            .Select(iter => iter.Iteration[metric.Name]);
+
+                        var count = values.Count();
+                        if (count == 0) // Cannot compute statistics when there are not results (e.g. user only ran a subset of all tests).
+                            continue;
+
+                        var avg = values.Average();
+                        var stdev_s = Math.Sqrt(values.Sum(x => Math.Pow(x - avg, 2)) / (count - 1));
+                        var max = values.Max();
+                        var min = values.Min();
+
+                        var newRow = dt.AppendRow();
+                        newRow[col0_testName] = test.Name;
+                        newRow[col1_metric] = metric.DisplayName;
+
+                        newRow[col2_iterations] = count.ToString();
+                        newRow[col3_average] = avg.ToString();
+                        newRow[col4_stdevs] = stdev_s.ToString();
+                        newRow[col5_min] = min.ToString();
+                        newRow[col6_max] = max.ToString();
+                    }
+                }
+            }
+
+            return dt;
+        }
+    }
+
+    
+    [Serializable]
+    [XmlType("NetcoreAssembly")]
+    public sealed class NetcoreAssembly
+    {
+        [XmlAttribute("Name")]
+        public string Name { get; set; }
+
+        [XmlElement("Tests")]
+        public List<NetcoreAssemblyTest> Tests { get; set; }
+
+        public NetcoreAssembly()
+        {
+            Tests = new List<NetcoreAssemblyTest>();
+        }
+
+        public NetcoreAssembly(string name)
+        {
+            Name = name;
+            Tests = new List<NetcoreAssemblyTest>();
+        }
+    }
+
+    [Serializable]
+    [XmlType("Test")]
+    public sealed class NetcoreAssemblyTest
```

Comment:
I think we probably want to call this class "TestModel" or something like that - the XML type can remain as is.  Same thing here - nothing .NET core specific.